src/test/java/com/wikia/webdriver/common/enums/Mercury.java

```diff
@@ -0,0 +1,31 @@
+package com.wikia.webdriver.common.enums;
+
+/**
+ * @ownership Content X-Wing Wikia
+ */
+public class Mercury {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1118)